docs/operational_guide/resource_limits.md

```diff
@@ -11,27 +11,50 @@ performance of M3 in a production environment.
 The best way to get started protecting M3DB nodes is to set a few limits on the
 top level `limits` config stanza for M3DB.
 
-When using M3DB for metrics workloads, queries arrive as a set of matchers 
-that select time series based on certain dimensions. The primary mechanism to 
-protect against these matchers matching huge amounts of data in an unbounded 
-way is to set a maximum limit for the amount of time series blocks allowed to
-be matched and consequently read in a given time window. This can be done using 
-`maxRecentlyQueriedSeriesBlocks` to set a maximum value and lookback time window 
-to determine the duration over which the max limit is enforced.
+The primary concern is to set a limit on the historical bytes read from disk 
```

Comment:
Maybe instead of "concern" say "strategy"?

Or rephased to say "The primary limit is on historical bytes read from disk..."